app/src/main/res/layout/group_find_references_in_page.xml

```diff
@@ -10,8 +10,8 @@
     <TextView
         android:id="@+id/reference_label"
         android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_gravity="center_vertical"
+        android:layout_height="48dp"
+        android:gravity="center_vertical"
         android:layout_marginStart="16dp"
```

Comment:
Looks like the text does not align with the count text.

Would it be better if we do something like this:
```
        android:layout_height="wrap_content"
        android:layout_gravity="center_vertical"
        android:padding="16dp"
        android:background="?android:attr/selectableItemBackground"
```
Set padding, change height to `wrap_content`, change to the previous gravity setting and add background of clicking ripple animation.